### PR TITLE
Align content with prototype

### PR DESCRIPTION
**Pull request by tvararu**
### Context

The start page content and service name is different.

### Changes proposed in this pull request

I've updated the content, added more translation strings, and tweaked some other small bits.

### Guidance to review

Review commit by commit if you need more context on some of the changes.

I've created a `locales/candidate.yml` locale file where we could chuck in most of the content for the candidate flow? Not sure what the best pattern is for working with large amounts of translations.

### Before

before

### After

Screenshot 2019-09-25 at 15 58 04

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -31,14 +31,17 @@
 
 en:
   page_titles:
-    application: 'Apply for postgraduate teacher training - GOV.UK'
-    welcome: 'Welcome'
+    application: "Apply for teacher training - GOV.UK"
```
Comment (alexdesi) [inline config/locales/en.yml]: use a single quote, or even better, no quotes

File: app/views/candidate_interface/start_page/show.html.erb [hunk 2]
```diff
@@ -3,18 +3,40 @@
...
+      <%= t('start.to_be_eligible') %>
+    </p>
+
     <ul class="govuk-list govuk-list--bullet">
-      <li>apply to your chosen course and other providers who have opted in to the trial</li>
-      <li>be enrolled in the same way as any other candidate, if you're successful</li>
-      <li>get support from the Department for Education (DfE) when you apply</li>
-      <li>tailor your application to suit different providers</li>
+      <li><%= t('start.a_degree') %></li>
+      <li><%= t('start.grade_4_english_maths') %></li>
```
Comment (alexdesi) [inline app/views/candidate_interface/start_page/show.html.erb]: unnecessary empty rows (there are others in the same file)

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -43,9 +43,11 @@
         </div>
         <div class="govuk-header__content">
           <a href="/" class="govuk-header__link govuk-header__link--service-name">
-            Apply for postgraduate teacher training
+            <%= t('layout.service_name') %>
           </a>
-          <p><%= current_candidate.email_address unless current_candidate.nil? %></p>
+          <% if current_candidate.present? %>
```
Comment (chubberlisk) [inline app/views/layouts/application.html.erb]:
```suggestion
          <% if candidate_signed_in? %>
```

Thinking we could the Devise's helper for checking a candidate is signed in?

File: config/locales/candidate.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+en:
+  start:
+    heading: Apply for teacher training
+    what_is_it: Apply for teacher training is a new GOV.UK service being trialled with a small number of training providers.
+    what_you_will_need: What you’ll need
+    to_be_eligible: "To be eligible for teacher training, you need these qualifications (or their non-UK equivalents):"
+    a_degree: a degree
+    grade_4_english_maths: grade 4 (C) or above in English and maths GCSEs
+    grade_4_science: grade 4 (C) or above in GCSE science if you want to teach primary
+    before_you_start: Before you start a teacher training course you’ll also need to confirm you have the health and physical capacity to start training.
```
Comment (alexdesi) [inline config/locales/candidate.yml]: should we group the items which are used in the radio button inside a `select:`?
```
  select:
    a_degree: a degree
    grade_4_english_maths: grade 4 (C) or above in English and maths GCSEs
    ...
```

File: app/views/candidate_interface/start_page/show.html.erb [hunk 5]
```diff
@@ -3,18 +3,40 @@
...
+      <li><%= t('start.grade_4_science') %></li>
     </ul>
 
-    <%= link_to candidate_interface_sign_up_path, role: 'button', class: 'govuk-button govuk-button--start', 'data-module': 'govuk-button' do %>
+    <p class="govuk-body">
+      <%= t('start.before_you_start') %>
```
Comment (alexdesi) [inline app/views/candidate_interface/start_page/show.html.erb]: empty row

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -31,9 +31,9 @@
...
...3 7.7-3.9 9.1zM69.7 17.7l8.9 4.7V9.3l-8.9 2.8c-.2-.3-.5-.6-.9-.9L72.4 0H59.6l3.5 11.2c-.3.3-.6.5-.9.9l-8.8-2.8v13.1l8.8-4.7c.3.3.6.7.9.9l-5 15.4v.1c-.2.8-.4 1.6-.4 2.4 0 4.1 3.1 7.5 7 8.1h.2c.3 0 .7.1 1 .1.4 0 .7 0 1-.1h.2c4-.6 7.1-4.1 7.1-8.1 0-.8-.1-1.7-.4-2.4V34l-5.1-15.4c.4-.2.7-.6 1-.9zM66 92.8c16.9 0 32.8 1.1 47.1 3.2 4-16.9 8.9-26.7 14-33.5l-9.6-3.4c1 4.9 1.1 7.2 0 10.2-1.5-1.4-3-4.3-4.2-8.7L108.6 76c2.8-2 5-3.2 7.5-3.3-4.4 9.4-10 11.9-13.6 11.2-4.3-.8-6.3-4.6-5.6-7.9 1-4.7 5.7-5.9 8-.5 4.3-8.7-3-11.4-7.6-8.8 7.1-7.2 7.9-13.5 2.1-21.1-8 6.1-8.1 12.3-4.5 20.8-4.7-5.4-12.1-2.5-9.5 6.2 3.4-5.2 7.9-2 7.2 3.1-.6 4.3-6.4 7.8-13.5 7.2-10.3-.9-10.9-8-11.2-13.8 2.5-.5 7.1 1.8 11 7.3L80.2 60c-4.1 4.4-8 5.3-12.3 5.4 1.4-4.4 8-11.6 8-11.6H55.5s6.4 7.2 7.9 11.6c-4.2-.1-8-1-12.3-5.4l1.4 16.4c3.9-5.5 8.5-7.7 10.9-7.3-.3 5.8-.9 12.8-11.1 13.8-7.2.6-12.9-2.9-13.5-7.2-.7-5 3.8-8.3 7.1-3.1 2.7-8.7-4.6-11.6-9.4-6.2 3.7-8.5 3.6-14.7-4.6-20.8-5.8 7.6-5 13.9 2.2 21.1-4.7-2.6-11.9.1-7.7 8.8 2.3-5.5 7.1-4.2 8.1.5.7 3.3-1.3 7.1-5.7 7.9-3.5.7-9-1.8-13.5-11.2 2.5.1 4.7 1.3 7.5 3.3l-4.7-15.4c-1.2 4.4-2.7 7.2-4.3 8.7-1.1-3-.9-5.3 0-10.2l-9.5 3.4c5 6.9 9.9 16.7 14 33.5 14.8-2.1 30.8-3.2 47.7-3.2z"></path>
```
Comment (alexdesi) [inline app/views/layouts/application.html.erb]: probably we should remove this `path` from the view (maybe not now anyway)

**Comment by tvararu** [comment]
Thanks for the reviews y'all; after reflection and some offline discussion I think it's now a better idea not to use translated strings for everything so will update this accordingly.

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (tvararu) [reply app/views/layouts/application.html.erb]: ✅

**Pull request merged**